JetAnalyzers/BuildFile.xml

```diff
@@ -23,9 +23,9 @@
 <use   name="rootcore"/>
 <use   name="rootMinuit"/>
 <use   name="rootspectrum"/>
-<flags   EDM_PLUGIN="1"/>
 <!--
+<flags   EDM_PLUGIN="1"/>
+-->
```

Comment:
Why the change from EDM_PLUGIN to lib? Will this effect the EDAnalyzer in this package?